### PR TITLE
Update to getting started guide for fine-3

### PR DESCRIPTION
**Pull request by JPrause**
[skip ci]

This is based on github issue: https://github.com/ManageIQ/manageiq.org/issues/543

### CODE REVIEW

**Comment by JPrause** [comment]
@cybette this is to resolve: https://github.com/ManageIQ/manageiq.org/issues/543
Please review and merge is all is good.

**Comment by cybette** [comment]
@JPrause looks great! Can you double check if screenshot0003.png and screenshot0004.png should be swapped but otherwise everything else checks out.

**Comment by JPrause** [comment]
Thanks @cybette it looks like the existing screen shots on the website (and their descriptions) are also a little off. I'll update the wording to more closely match the updated screen shots.

File: site/docs/get-started/basic-configuration.md [hunk 1]
```diff
@@ -31,31 +31,31 @@ The main menus are available in
...
--         The *hostname* and *IP address* of the appliance will be shown here, be careful as in some
-        cases this will be the internal IP address and not the one you
-        connected to.  
-
--         You can change the *Company Name* to whatever you want, the change will be reflected in the interface.  ***Change it to ManageIQ***
-
--         ***Select your time zone***, and,
-        if you want, you can change the language default.
+-       ***CFME Region: Region 0*** → ***Zones*** → ***Zone: Default zone(current)*** →
+        ***Server: EVM\[1\](current)*** should be selected by default.
```
Comment (cybette) [inline site/docs/get-started/basic-configuration.md]: I think that's too many unnecessary asterisks. Might be good to clean it up like:

-       ***CFME Region: Region 0 → Zones → Zone: Default zone(current) →
        Server: EVM\[1\](current)*** should be selected by default.

**Comment by cybette** [comment]
Near the end of the doc,

`**Press Save*** to save your changes` - either one missing asterisk to the left or one extra on the right, depending on whether you want italic bold, or just bold :)

**Pull request merged**